PRIORITY_NOTES.md

```diff
@@ -0,0 +1,17 @@
+This file contains some notes on the Jenkins job priorities that are configured in this repository.
+
+The ROS buildfarm is currently setup with a Jenkins plugin that manages job priorities.  The priorities are inverse numerically based; that is, a lower number means a higher priority.  As of this writing, the buildfarm is configured to have a range of 1 to 109.  There are periodic maintenance jobs that use priorities 1 to 49, thus those priorities should never be used for distribution jobs.
+
+There are currently 6 classes of distribution jobs that the buildfarm runs.  Within each class, the ROS distributions should be in release order (newest ROS release has highest priority).  To make it so that we don't have to change the priorities on every release, we have defined some ranges for the classes, listed in the table below:
+
+| Job Type                     | Numeric Range |
+| ---------------------------- |:-------------:|
+| Pull Request                 | 40 - 49       |
+| Commit                       | 50 - 59       |
+| Source                       | 60 - 69       |
+| Basic Binary (amd64)         | 70 - 79       |
```

Comment:
Nit: this applies to any non-ARM job at the moment (x86 and amd64)

Reply:
> Nit: this applies to any non-ARM job at the moment (x86 and amd64)

Didn't we drop support for x86 a while back, though?

Reply:
In Lunar, yes. But Indigo and Kinetic are still building x86.

Reply:
Ah, got it.  Thanks for the clarification.